snooty/rstspec.toml

```diff
@@ -1011,13 +1011,25 @@ help = """Link to a page in the MongoDB Guides site."""
 type = {link = "https://docs.mongodb.com/guides%s"}
 
 [role.mms-docs]
-help = """Link to a page in the MongoDB Guides site."""
+help = """Link to a page in the Cloud Manager documentation."""
 type = {link = "https://docs.cloud.mongodb.com%s"}
 
 [role.mms-home]
 help = """Link to a MongoDB Cloud Services page."""
 type = {link = "https://cloud.mongodb.com%s"}
 
+[role.osb]
+help = """Link to a page in the Atlas Open Service Broker documentation."""
+type = {link = "https://docs.mongodb.com/atlas-open-service-broker/current%s"}
+
+[role.adl]
+help = """Link to a page in the Atlas Data Lake documentation."""
+type = {link = "https://docs.mongodb.com/datalake%s"}
+
+[role.dochub]
+help = """Specify a dochub link."""
+type = {link = "https://dochub.mongodb.org/core/%s"}
```

Comment:
Just curious: are we including some trailing slashes and omitting others in order to replicate the legacy setup?